packages/oasis-actions/src/strategies/aave/open/open.ts

```diff
@@ -137,7 +148,7 @@ export async function open(
 
 async function simulatePositionTransition(
   quoteSwapData: SwapData,
-  args: AaveOpenArgs,
+  args: AaveOpenArgs & { fee: BigNumber },
```

Comment:
Maybe we can create a type:
```
type WithFee = { fee: BigNumber }
```

Or even:
```
type WithFee = { amount: BigNumber; token: string }
```
Then we can return `IPositionTranstion & WithFee` because the fee is now bounded with the Swap. It could be a betterod approach because, on Close To Collateral, we take fee after payback debt.